tests/components/device/ace.spec.ts

```diff
@@ -556,10 +564,49 @@ describe('Ace', () => {
       }
     });
   });
+  describe('#getCnnFeatureStatus', () => {
+    sinon.spy(dummyTransport.grpcClient, 'getCnnFeatureStatus');
+    const cnnFeature = new huddly.CnnFeature();
+    cnnFeature.setFeature(huddly.Feature.AUTOZOOM);
```

Comment:
Shouldnt this go into a `BeforeEach` block?

Reply:
Yes, you're right. Think the whole test file needs a bit of an overhaul. Will fix this one and then go over the rest tomorrow morning :)